### PR TITLE
v0.2.0

### PR DESCRIPTION
**Pull request by noamross**
### New functionality

- Add a FASTA export function: `ed_fasta()`
- Add a function for reports on recents tests: `ed_tests_report()`

### Changes to data processing

 - New column in tests table - `diag_lab_shortname` - an abbreviated lab name for easier reporting
 - Renamed `known_human_risk` to `evidence_human_infection` and added more clarifying text to metadata
 - Taxagroups are now one-to-one mapped to orders. `ed_taxagroups()` will return a table with the
   mapping between them.  Metadata notes updated to reflect this. (See [Issue #32](https://github.com/ecohealthalliance/eidith/issues/32))
 
### Bug Fixes

 - No longer fail when database extracts are zero-length
 
### Changes to development

 - Moved to separate stable (`master`) and development (`dev`) branches. Stable
   branch can now be installed without devtools.
 - Set up continuous integration tests, run daily, for all main OSs and R versions. These will tell us if patches or new data break something in the package.

### CODE REVIEW

File: tests/testthat.R [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
 library(eidith)
 
+
+test_that("ed_tests_report is returning an error", {
+  expect_error(ed_tests_report(), NA)
+}
+)
+
+
 test_check("eidith")
```
Comment (noamross) [inline tests/testthat.R]: This test should go in a new file under `tests/testthat` Note the filename structure there, as well as the "if" statements in those files. In tests, first everything in the helpers file is run, then tests are run consecutively.  Things requiring a database to exist have to go after the first set of tests, when data are downloaded, and one should use an if statement to only run the test is there is a database to test against.

Comment (calebasaraba) [inline tests/testthat.R]: Ahh got it!

**Comment by noamross** [comment]
@calebasaraba, Check if you are still getting that same error you had above? I can't reproduce it, suspect that it's due to your database not being updated after you installed this version.

**Comment by calebasaraba** [comment]
You are 100% correct, I was updating in the wrong order. ed_tests_report() and ed_fasta() are now working for me.

**Pull request merged**